Pod/Classes/DateConvenience.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  DateConvenience.swift
+//  Pods
+//
+//  Created by Jason Grandelli on 11/14/16.
+//
+//
+
+import Foundation
+
+public extension Date {
+    // MARK: Units
+    public var components: DateComponents? {
+        return Calendar.current.dateComponents([.era, .year, .month, .day, .hour, .minute, .second, .weekday, .weekdayOrdinal, .quarter, .weekOfMonth, .weekOfYear, .yearForWeekOfYear, .nanosecond, .calendar, .timeZone], from: self)
+    }
+
+    public static func monthSymbols() -> [String] {
+        return DateFormatter().monthSymbols
+    }
+    
+    public static func monthSymbol(month: Int) -> String? {
+        guard month < monthSymbols().count else { return nil }
+        
+        return monthSymbols()[month]
```

Comment:
i would assume so, most of this is largely copied from mt_dates, so it will work however that would work. but in general it depends on the calendar of course